### DISCUSSION TITLE
How this work

### DISCUSSION THREAD

**Discussion by Missy45** [community/community#142535] [New to GitHub]
I don't know how to do any of this so if someone can please help me and let me know if I in the right place or this is not where I suppose to be help me please missy45

**Comment by ghostinhershell** [comment]
Hi @Missy45, we're happy you're here! You are more likely to get a useful response if you are explicit about what your project or issue entails. Giving a few more details, or asking a clearer question, might help someone give you a nudge in the right direction.

**Reply by aonez** [reply]
Maybe it’s bot/spam?
https://github.com/aonez/Keka/discussions/1538
